england.qmd

```diff
@@ -0,0 +1,492 @@
+  labs(
+    x = NULL, y = NULL,
+    title = paste0("Life expectancy by practice, birth cohort 2016-20 (England)"),
+    subtitle = "Practices in most and least deprived IMD quintiles. Larger circles represent stronger socio-economic gradient.",
+    colour = "",
+    caption = "Source: NHS England GP Workforce statistics, 2024. © Health Equity Evidence Centre, 2024."
+  ) +
+  coord_flip() +
+  guides(colour = guide_legend(override.aes = list(size = 50, shape = c(16, 16, 18))))
+```
+
+Average life expectancy for men is **`r round(df_filter[df_filter$Indicator == "Life_Expectancy_Male", ]$quin_1, 1)`** in the least deprived 20% and **`r round(df_filter[df_filter$Indicator == "Life_Expectancy_Male", ]$quin_5, 1)`** in the most deprived 20%.
+
+Average life expectancy for women is **`r round(df_filter[df_filter$Indicator == "Life_Expectancy_Female", ]$quin_1, 1)`** in the least deprived 20% and **`r round(df_filter[df_filter$Indicator == "Life_Expectancy_Female", ]$quin_5, 1)`** in the most deprived 20%.
+
+## Inequality in NHS Payments
+
+```{r payments}
+year <- 2023
+
+df_filter <- df %>%
+  filter(ICB.NAME == "England") %>%
+  filter(Year == year) %>%
+  filter(Indicator %in% c("payment_per_patient_all", "payment_per_patient_disp", "payment_per_patient_non_disp")) %>%
+  mutate(Indicator = factor(Indicator, levels = c("payment_per_patient_disp", "payment_per_patient_non_disp", "payment_per_patient_all")))
+
+df_filter %>%
+  ggplot(., aes(x = Indicator)) +
+  geom_linerange(aes(x = Indicator, ymin = quin_5, ymax = quin_1)) +
+  geom_point(aes(y = quin_1, color = "Least deprived")) +
+  geom_point(aes(y = quin_5, color = "Most deprived")) +
+  geom_point(aes(y = avg, color = "National average"), size = 15, shape = 18) +
+  scale_x_discrete(labels = c(
+    "payment_per_patient_all" = "All practices",
+    "payment_per_patient_disp" = "Dispensing practices",
+    "payment_per_patient_non_disp" = "Non-dispensing practices"
+  )) +
+  scale_color_manual(values = c("Least deprived" = "#009639", "Most deprived" = "#003087", "National average" = "#DA291C")) +
+  labs(
+    x = NULL, y = NULL,
+    title = paste0("Mean payment per weighted patient, 2022/23 (England)"),
+    subtitle = "Practices in most and least deprived IMD quintiles",
+    colour = "",
+    caption = "Source: NHS England Payments to General Practice, 2022/23. © Health Equity Evidence Centre, 2024."
+  ) +
+  coord_flip() +
+  guides(colour = guide_legend(override.aes = list(size = 50, shape = c(16, 16, 18))))
+```
+
+Average payment per weighted patient is **£`r round(df_filter[df_filter$Indicator == "payment_per_patient_all", ]$quin_5, 2)`** in the most deprived 20% of practices, versus **£`r round(df_filter[df_filter$Indicator == "payment_per_patient_all", ]$quin_1, 2)`** in the least deprived 20%.
+
+If there are no dispensing practices in the most deprived quintile, we use the second most deprived. If both are missing, we omit the data point from the figure.
```

Comment:
I assume this relates to when the data is used for other areas as these charts are for England and so there is data at these quintiles. Could this be amended to advise this is guidance as the code doesn't include this logic. I'm assuming that readers of this may well be new to R and want to follow the same principles.